### PR TITLE
hunspell: add page

### PR DESCRIPTION
**Pull request by newAM**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/hunspell.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# hunspell
+
+> Check spelling.
+> More information: <http://hunspell.github.io/>.
```
Comment (navarroaxel) [inline pages/common/hunspell.md:4]:
```suggestion
> More information: <https://hunspell.github.io>.
```

File: pages/common/hunspell.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Check the spelling of a file:
+
+`hunspell {{file}}`
```
Comment (navarroaxel) [inline pages/common/hunspell.md:8]:
```suggestion
`hunspell {{path/to/file}}`
```

File: pages/common/hunspell.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Check the spelling of a file with the en_US dictionary:
+
+`hunspell -d en_US {{file}}`
```
Comment (navarroaxel) [inline pages/common/hunspell.md:12]:
```suggestion
`hunspell -d {{en_US}} {{path/to/file}}`
```

File: pages/common/hunspell.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- List bad words in a file:
+
+`hunspell -l {{file}}`
```
Comment (navarroaxel) [inline pages/common/hunspell.md:16]:
```suggestion
`hunspell -l {{path/to/file}}`
```

File: pages/common/hunspell.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Check the spelling of a file:
+
+`hunspell {{file}}`
+
+- Check the spelling of a file with the en_US dictionary:
+
+`hunspell -d en_US {{file}}`
+
+- List bad words in a file:
```
Comment (navarroaxel) [inline pages/common/hunspell.md:14]:
```suggestion
- List misspelled words in a file:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @newAM, welcome and thanks for your submission. I left some comments.

**Comment by newAM** [comment]
Made the suggested changes :+1:

**Review by owenvoke — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**